web-marketplace/src/components/organisms/Order/Order.tsx

```diff
@@ -37,107 +41,178 @@ export const Order = ({ orderData, allowedDenoms, className }: OrderProps) => {
-                  ? _(msg`Expected delivery date`)
-                  : ''
-              } ${project.deliveryDate}`}</p>
-            </div>
-          </>
-        }
-      />
-      <CardContent className="p-15 pt-40 sm:p-30 bg-grey-0">
-        <OrderSummary
-          retirementInfo={retirementInfo}
-          blockchainDetails={blockchainDetails}
-          creditsData={credits}
-          paymentInfo={paymentInfo}
-          allowedDenoms={allowedDenoms}
+          }
+          title={
+            <Title
+              variant="h2"
+              className="mt-10 sm:mt-0 mb-5 text-[21px] leading-[1.5]"
+            >
+              <Link href={projectHref} className="text-bc-neutral-700">
+                {project.name}
+              </Link>
+            </Title>
+          }
+          subheader={
+            <>
+              <div className="flex items-center my-5">
+                <ProjectPlaceInfo
+                  place={project.place}
+                  area={project.area}
+                  iconClassName="mr-5"
+                  areaUnit={getAreaUnit(
+                    project.areaUnit as qudtUnit | undefined,
+                    project.area,
+                  )}
+                  fontSize="13px"
+                  color={theme.palette.primary.contrastText}
+                />
+              </div>
+              <div className="flex flex-row items-center mb-20 md:mb-0">
+                <OrderLabel
+                  type={status}
+                  // eslint-disable-next-line lingui/no-unlocalized-strings
+                  wrapperClassName="place-self-start md:place-self-center"
+                />
+                <p className="m-0 ml-5 text-xs text-grey-700">{`${
+                  status === ORDER_STATUS.pending
+                    ? _(msg`Expected delivery date`)
+                    : ''
+                } ${project.deliveryDate}`}</p>
+              </div>
+            </>
+          }
         />
-      </CardContent>
-    </Card>
+        <CardContent className="p-15 pt-40 sm:p-30 bg-grey-0">
+          <OrderSummary
+            retirementInfo={retirementInfo}
+            blockchainDetails={blockchainDetails}
+            creditsData={credits}
+            paymentInfo={paymentInfo}
+            displayDenom={displayDenom}
+            displayTotalPrice={displayTotalPrice}
+            currency={currency}
+          />
+        </CardContent>
+      </Card>
+      {cryptoReceiptModalState.open && (
+        <OrderCryptoReceiptModal
+          onClose={setCryptoReceiptModalState}
+          open={cryptoReceiptModalState.open}
+          modalContent={{
+            displayDenom,
+            currency,
+            price: displayTotalPrice,
+            project: {
+              name: project.name,
+              projectHref,
+            },
+            amount: credits.credits,
+            date: blockchainDetails.purchaseDate,
+            txType: cryptoReceiptModalState.type,
```

Comment:
not sure if `txType` is the right term here, it's not really about the tx but rather about the credits being retired or not
also we could simply provide `retirementInfo.retiredCredits` instead of getting it from `cryptoReceiptModalState`, and then based on `retiredCredits` bool value we display the appropriate content in `OrderCryptoReceiptModal`